files/zh-cn/web/html/element/footer/index.md

```diff
@@ -1,37 +1,112 @@
 ---
-title: <footer>
+title: <footer>：页脚元素
 slug: Web/HTML/Element/footer
+l10n:
+  sourceCommit: fdd3ac5598c3ddceb71e59949b003936ae99f647
 ---
 
 {{HTMLSidebar}}
 
-**HTML \<footer> 元素**表示最近一个[章节内容](/zh-CN/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document#Defining_Sections_in_HTML5)或者[根节点](/zh-CN/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document#Sectioning_root)（sectioning root）元素的页脚。一个页脚通常包含该章节作者、版权数据或者与文档相关的链接等信息。
+[HTML](/zh-CN/docs/Web/HTML) **`<footer>`** 元素表示其最近的祖先[分段内容](/zh-CN/docs/Web/HTML/Content_categories#分段内容)的页脚或[分段内容](/zh-CN/docs/Web/HTML/Element/Heading_Elements#分段内容)元素。`<footer>` 通常包含有关该部分作者、版权数据或相关文档链接的信息。
```

Comment:
The [en-US document](https://github.com/mdn/content/blob/5671055d63552c5a4dc22ce7f6bea408afa1521a/files/en-us/web/html/element/footer/index.md?plain=1#L10) uses term "section root", and the link to target document is broken, please fix this.

Reply:
```suggestion
[HTML](/zh-CN/docs/Web/HTML) **`<footer>`** 元素表示其最近的祖先[分段内容](/zh-CN/docs/Web/HTML/Content_categories#分段内容)的页脚或[分段根](/zh-CN/docs/Web/HTML/Element/Heading_Elements#标注章节内容)元素。`<footer>` 通常包含有关该部分作者、版权数据或相关文档链接的信息。
```

Reply:
I mean: "section root" section is not existed in [Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) 🤕

Reply:
> I mean: "section root" section is not existed in [Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) 🤕

Oops, I've raised https://github.com/mdn/content/pull/32817 to fix it